ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/StandardJsonTracer.java

```diff
@@ -128,14 +128,13 @@ public void traceExecution(
 
   private Gas computeCallGas(
       final AbstractCallOperation callOperation, final MessageFrame messageFrame) {
-    final GasCalculator gasCalculator = callOperation.gasCalculator();
     // as part of EIP 150 the returned costGas is gas - base * 63 / 64.
```

Comment:
this is non-blocking, but should we change this comment also?  it isn't base cost * 63/64  but calculated operation cost * 63/64, right?